scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -251,19 +279,20 @@ object Measurer {
       }
 
       var best = results
-      while (outlierExists(results) && retleft > 0) {
-        val prefixlen = suffixLength(results.reverse)
-        val suffixlen = suffixLength(results)
-        val formatted = results.map(t => f"$t%.3f")
+      while (outlierExists(results.map(_.value.toDouble())) && retleft > 0) {
+        val prefixlen = suffixLength(results.reverse.map(_.value.toDouble()))
+        val suffixlen = suffixLength(results.map(_.value.toDouble()))
+        val formatted = results.map(t => f"${t.value.toDouble()}%.3f")
```

Comment:
Well, if this generic param is `Double` and those `Double`  have a lot of digit after decimal point then the printing won't be so readable.
On the other hand, yes we unnecessarily also convert `Int`s and so on to `Double`.
So I don't know which choice is better.
Alternatively, I guess that some pattern matching here could be an option.